### PR TITLE
Display asset info in lot registry

### PR DESCRIPTION
**Pull request by jmcameron**
Allow the display of asset info in the lot registry.  Also support editing the asset serial number in the lot editor modal.

Finishes items in Issue https://github.com/IMA-WorldHealth/bhima/issues/6349

**TESTING**
- Use any dataset
- Use the Stock > articles in Stock to find an inventory item with articles in stock.
- Edit the inventory item for the lot from the last step and change it to an asset.  Fill out the other asset related fields (manufacturer brand, model, etc).  Then save
- Go back to the lots in stock for the inventory item modified above
- Use [Menu] > Change Columns to display the asset related info (manufacturers info, serial number, etc).
- Verify that the asset-related fields are shown in the lot registry grid display
- Use the action menu to edit and save the asset and set its serial number
- Verify that the serial number just added is displayed.   
- Edit the asset again and change the serial number.  Verify it is changed in the grid.
- Revert the inventory item to it prior 'asset' status (or rebuild the database)

### CODE REVIEW

File: client/src/modules/stock/lots/registry.service.js [hunk 1]
```diff
@@ -187,6 +187,37 @@ function LotsRegistryService(uiGridConstants, Session) {
...
+      headerCellFilter : 'translate',
+      visible : false,
+      cellTemplate : '/modules/inventory/list/templates/asset.cell.tmpl.html',
+    }, {
+      field : 'reference_number',
+      displayName : 'FORM.LABELS.REFERENCE_NUMBER',
+      headerCellFilter : 'translate',
+      visible : false,
+      width : 150,
+      // ??? cellTemplate : '/modules/inventory/list/templates/reference_number.cell.html',
```
Comment (mbayopanda) [inline client/src/modules/stock/lots/registry.service.js]: You can remove this line I think the current way is okay without this template.

**Review by mbayopanda — APPROVED** [review]
LGTM 👍🏽 

The PR is fine to be merged just a minor comment about this commented line.

File: client/src/modules/stock/lots/registry.service.js (hunk 1 above)
Comment (jmcameron) [reply client/src/modules/stock/lots/registry.service.js]: Done

**Comment by jniles** [comment]
Looks like this got approval and the issue was fixed.  Merging.

bors r+

**Pull request merged**